docs/modules/commons/pages/cli.adoc

```diff
@@ -0,0 +1,298 @@
+= CLI
+
+== Run stories
+
+Task performs test configuration corectness and spotless checks and then runs test stories.
```

Comment:
it's not needed to describe all the stages, just the main purpose. Also pay attention to the articles

Reply:
here and below